.github/workflows/autoformat.yml

```diff
@@ -54,6 +53,11 @@ jobs:
 
   php-cs-fixer:
     runs-on: ubuntu-latest
+
+    permissions:
+      # Give the default GITHUB_TOKEN write permission to commit and push the changed files back to the repository.
+      contents: write
```

Comment:
Awesome!

Reply:
I went through readme, hope it works as expected.